app/background.js

```diff
@@ -29,7 +29,11 @@ const checkToClose = function(cutOff) {
   }
 
   // Update the selected one to make sure it doesn't get closed.
-  chrome.tabs.query({active: true}, tabmanager.updateLastAccessed);
+  chrome.tabs.query({active: true, currentWindow: true}, tabmanager.updateLastAccessed);
```

Comment:
My understanding from reading Chrome's definition of [the current window](https://developer.chrome.com/extensions/windows#current-window) is that it is not necessarily the window focused by the user. I believe you should instead query for `lastFocusedWindow: true`.

Can you test `lastFocusedWindow` to ensure it works as expected?

Reply:
`lastFocusedWindow` seems to work as expected. I have added a commit to reflect these changes. 

I'm still learning the ropes with Git and my second commit was meant as a separate pull request. If you wish I will attempt to undo it and recommit on a different branch.

Reply:
Thanks for looking into `lastFocusedWindow`!

I'm fine merging this whole pull request, but if you want some practice you can try splitting out that commit. One option is something like this:

```
git reset --soft HEAD~2
git add app/background.js
git commit -m "change currentWindow to lastFocusedWindow"
git push -f
```

that'll put only the changes for this pull request into this branch. Documentation will tell you "push --force" is evil, which is true if you are working with other developers. If you are the only contributor to a branch though, it's fine and simpler than other approaches.

Then you can create a new branch for your other feature (this is assuming you used the commands above and have uncommitted local changes):

```
git checkout master
git checkout -b dont-close-audio-tabs
git add .
git commit -m "Added option to prevent tabs that play audio from getting closed"
git push -u origin dont-close-audio-tabs
```

Reply:
Thanks for the guide. I had a few struggles getting this to work exactly and I don't want to accidentally mess things up, so I will leave it like it is.

Also, the merge conflict should be resolved now.